### ISSUE TITLE
paper dialog open function

### ISSUE THREAD

**Issue by lilletech** [vaadin/gwt-polymer-elements#12]
when i use a paper dialog, the first call to the open method giving an exception. (after the calls works)

Uncaught com.google.gwt.event.shared.UmbrellaException: Exception caught: (TypeError) : this.getPolymerElement_11_g$(...).open is not a functionqtc_g$ @ HandlerManager.java:129Y6c_g$ @ Widget.java:129Hjc_g$ @ DomEvent.java:125e7c_g$ @ Widget.java:177Brd_g$ @ DOM.java:1480Ard_g$ @ DOM.java:1419Sxd_g$ @ DOMImplStandard.java:317fU_g$ @ Impl.java:247iU_g$ @ Impl.java:306(anonymous function) @ Impl.java:72

PaperButton b = new PaperButton("button");
b.setCenter(true);
add(b);

b.addClickHandler((ClickEvent event) -> {
  final PaperDialog d = new PaperDialog("Test");
  d.setTransition("core-transition-center");
  d.open();
});

BUT

when instanciation of dialog is not in the handler, only the first call to open works.

PaperButton b = new PaperButton("button");
    b.setCenter(true);
    add(b);

```
final PaperDialog d = new PaperDialog("Test");
d.setTransition("core-transition-center");
b.addClickHandler((ClickEvent event) -> {

  d.open(); // only the first click open the dialog.
});
```

**Comment by lilletech** [comment]
i think its a polymer bug  ->
https://www.polymer-project.org/0.5/components/paper-elements/demo.html#paper-dialog
it does the same

**Comment by lilletech** [comment]
the problem is resolved in the polymer version 1.0 !
https://elements.polymer-project.org/elements/paper-dialog?view=demo:demo/index.html

**Comment by manolo** [comment]
Nice to know that it's fixed in 1.0.
We are working on porting gwt-polymer to last version of polymer, I'll update the status when it is available.
